common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -178,6 +179,9 @@ private boolean handleUrl(final WebView view, final String url) {
             } else if (isAuthAppMFAUrl(formattedURL)) {
                 Logger.info(methodTag,"Request to link account with Authenticator.");
                 processAuthAppMFAUrl(url);
+            } else if (isAmazonAppRedirect(url)) {
```

Comment:
isAmazonAppRedirect(formattedURL)?

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2265/commits/8340bdadf0f692c496a708f737c088251aae542b